cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -118,3 +117,49 @@ def test_calls_child_block_is_submitted(self, mock_getstreamfields, mock_hasattr
         form_module_handlers(self.page, self.request, self.context)
         child.block.is_submitted.assert_called_with(self.request, 'name', 0)
 
+
+class TestRelativePageLinkHandler(TestCase):
+    def setUp(self):
+        self.default_site = Site.objects.get(is_default_site=True)
+
+    def test_nonexisting_page_returns_empty_link(self):
+        self.assertEqual(
+            RelativePageLinkHandler.expand_db_attributes(
+                attrs={'id': 0},
+                for_editor=False
+            ),
+            '<a>',
+        )
+
+    def test_page_not_in_site_root_returns_none_link(self):
+        self.assertEqual(
+            RelativePageLinkHandler.expand_db_attributes(
+                attrs={'id': 1},
+                for_editor=False
+            ),
+            '<a href="None">',
```

Comment:
A recent example that happened to us was when a live page contained a link to a "deleted" page that had been moved to the Trash. As the Trash isn't under a site root, it turned the live page's link into `<a href="None">`.